doc/api/punycode.md

```diff
@@ -1,6 +1,11 @@
 # punycode
 
-    Stability: 2 - Stable
+    Stability: 0 - Deprecated
+
+**The version of the punycode module bundled in Node.js is being deprecated**.
+In a future major version of Node.js this module will be removed. Users
```

Comment:
I'm not sure how specific we can be... Generally we've tried not to commit
to specific versions. It would be at least v8 before we could do the hard
deprecation.

On Monday, August 1, 2016, Mathias Bynens <notifications@github.com> wrote:

> In doc/api/punycode.md
> <https://github.com/nodejs/node/pull/7941#discussion_r73102561>:
>
> > @@ -1,6 +1,11 @@
> >  # punycode
> >
> > -    Stability: 2 - Stable
> > +    Stability: 0 - Deprecated
> > +
> > +**The version of the punycode module bundled in Node.js is being deprecated**.
> > +In a future major version of Node.js this module will be removed. Users
>
> Yeah, let’s be specific. I’ll then update the Punycode.js README
> accordingly.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7941/files/a1ee6dac91984047af0f6167031e6716bf6c8648#r73102561>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2ecTY2uR6HajrntHHGZHazWU51Yn7ks5qbunWgaJpZM4JZ4pZ>
> .
>